tests/bulk_create/tests.py

```diff
@@ -51,6 +54,30 @@ def test_inheritance(self):
             "Nicholas's",
         ], attrgetter("name"))
 
+    def test_proxy_inheritance(self):
+        """Ticket 24997: Support bulk_create on proxy models."""
```

Comment:
the ticket reference doesn't seem necessary since the expected behavior is straightforward